src/test/net_tests.cpp

```diff
@@ -105,7 +101,6 @@ BOOST_AUTO_TEST_CASE(cnode_listen_port)
 BOOST_AUTO_TEST_CASE(caddrdb_read)
 {
     CAddrManUncorrupted addrmanUncorrupted;
-    addrmanUncorrupted.MakeDeterministic();
```

Comment:
579584d Should the 4 CAddrMan ctors that follow pass deterministic `true` instead of `false`?

```diff
-    CAddrMan addrman1(/* deterministic */ false, /* consistency_checks */ 0);
+    CAddrMan addrman1(/* deterministic */ true, /* consistency_checks */ 0);
```

Reply:
I don't think so. They weren't deterministic before.